### PR TITLE
Add fresenius frontend

### PR DESCRIPTION
**Pull request by endverbraucher**
Introduce changes to add fresenius university customizations and some general changes for whitelabel products:

**general changes:**
* whitelabel stuff is moved to a seperate config file for the moment
* to have a better differentiation of the recover brand, the following changes are introduced:
  * whitelabels have no shifted background on checkboxes and the circle icon
  * background color is a more neutral grey

**for fresenius:**
* adds custom logo
* adds custom background color
* adds custom highlight color
* adds custom introtext

### CODE REVIEW

File: ui/layouts/MobileApp.tsx [hunk 1]
```diff
@@ -58,18 +64,20 @@ const LogoBox = styled(motion.div)(
...
+        maxWidth: '100%',
       },
       sticky: {
-        top: 8,
+        top: 4,
         position: 'fixed',
-        width: isCareEnv ? '92px' : '61px',
-        height: '10px',
+        width: logoSmallWidth,
+        height: logoSmallHeight,
```
Comment (salimhb) [inline ui/layouts/MobileApp.tsx:80]: Is it possible to center the logo or make it as wide as the white boxes below?
image

**Review by salimhb — APPROVED** [review]

File: pages/checkin.tsx [hunk 2]
```diff
@@ -168,20 +169,18 @@ export default function CheckinPage() {
...
                 : 'So kann das Gesundheitsamt Dich anrufen, wenn es notwendig ist.'}
             </p>
             <p>
-              Datenschutz ist uns dabei sehr wichtig! <strong>recover</strong>{' '}
-              speichert {isCareEnv ? 'Ihre' : 'Deine'} Daten verschlüsselt und
-              sicher.
+              Datenschutz ist uns dabei sehr wichtig!{' '}
+              {isRcvrEnv
+                ? <strong>recover</strong> +
+                  ' speichert Deine Daten verschlüsselt und sicher.'
```
Comment (danieldiekmeier) [inline pages/checkin.tsx:182]: image

You can't use `+` here.

```suggestion
                ? <><strong>recover</strong> speichert Deine Daten verschlüsselt und sicher.</>
```

File: ui/whitelabels.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    logoSmallWidth: '61px',
+    logoSmallHeight: '10px',
+    logoBigWidth: '182px',
+    logoBigHeight: '20px',
+    introText:
+      'Durch die aktuellen Corona-Verordnungen musst du deine Kontaktdaten hinterlegen, wenn Du in einem Betrieb bist der zu Schutzmaßnahmen verpflichtet ist, wie z.B Restaurants. Die App kann auch freiwillig genutzt werden, um die Nachverfolgung zu unterstützen.',
+  },
+  care: {
+    backgroundColor: '#f2f2f2',
+    highLightColor: '##F5B743',
```
Comment (danieldiekmeier) [inline ui/whitelabels.ts:19]: Whoops

```suggestion
    highlightColor: '#F5B743',
```

File: ui/theme.ts [hunk 4]
```diff
@@ -111,7 +112,7 @@ const globalStyles = css({
     fontSize: 16, // baseline for rems
     minHeight: '100vh',
     scrollBehavior: 'smooth',
-    bg: 'bluegrey.50',
+    bg: isRcvrEnv ? 'bluegrey.50' : backgroundColor,
```
Comment (danieldiekmeier) [inline ui/theme.ts:115]: Why do you use a ternary here instead of putting it into `whitelabels.ts` like for the other envs?

File: ui/theme.ts [hunk 5]
```diff
@@ -43,8 +44,8 @@ const theme = {
       900: '#090e10',
     },
     black: '#000000',
-    green: isCareEnv ? '#F5B743' : '#28EE5F',
-    pink: isCareEnv ? '#F5B743' : '#EA28EE',
+    green: isRcvrEnv ? '#28EE5F' : highlightColor,
+    pink: isRcvrEnv ? '#EA28EE' : highlightColor,
```
Comment (danieldiekmeier) [inline ui/theme.ts:48]: Maybe we can have `primaryHighlight` and `secondaryHighlight` so we can still put this into `whitelabel.ts`?

File: pages/checkin.tsx [hunk 6]
```diff
@@ -168,20 +169,18 @@ export default function CheckinPage() {
...
             <p>
-              {isCareEnv
-                ? 'Durch die aktuellen Corona-Verordnungen müssen Sie Ihre Kontaktdaten hinterlegen, wenn Sie in einem Betrieb sind der zu Schutzmaßnahmen verpflichtet ist, wie z.B Pflegeeinrichtungen. Die App kann auch freiwillig genutzt werden, um die Nachverfolgung zu unterstützen.'
-                : 'Durch die aktuellen Corona-Verordnungen musst du deine Kontaktdaten hinterlegen, wenn Du in einem Betrieb bist der zu Schutzmaßnahmen verpflichtet ist, wie z.B Restaurants. Die App kann auch freiwillig genutzt werden, um die Nachverfolgung zu unterstützen.'}
-            </p>
-            <p>
-              {isCareEnv
+              {isCareEnv || isFreseniusEnv
                 ? 'So kann das Gesundheitsamt Sie anrufen, wenn es notwendig ist.'
                 : 'So kann das Gesundheitsamt Dich anrufen, wenn es notwendig ist.'}
```
Comment (danieldiekmeier) [inline pages/checkin.tsx:176]: Maybe we can try to avoid all these ternaries by putting all the values into `whitelabels.ts`

**Review by danieldiekmeier — CHANGES_REQUESTED** [review]

File: pages/checkin.tsx (hunk 2 above)
Comment (endverbraucher) [reply pages/checkin.tsx]: I have no Idea why this happened. 🤔

File: ui/theme.ts (hunk 4 above)
Comment (endverbraucher) [reply ui/theme.ts:115]: Good question. Will change that.

File: ui/theme.ts (hunk 5 above)
Comment (endverbraucher) [reply ui/theme.ts:48]: Sure. I'll order some :)

File: pages/checkin.tsx (hunk 6 above)
Comment (endverbraucher) [reply pages/checkin.tsx:176]: Generally yes. In this case the text handles casual vs. more formal address of users. I think this should be handled by a setting in combination with I18n. I can change this so we have a setting for this in the whitelabels config. But I will not update the whole app to use this in this pr. We should do this when we implement a more solid way of handling whitelabels.

File: pages/business/index.tsx [hunk 7]
```diff
@@ -10,9 +10,7 @@ export default function BusinessIndexPage() {
   return (
     <MobileApp logoVariant="big">
       <Head>
-        <title key="title">
-          Für {isCareEnv ? 'Pflegeeinrichtungen' : 'Betriebe'} | recover
-        </title>
+        <title key="title">format</title>
```
Comment (danieldiekmeier) [inline pages/business/index.tsx:13]: `format`?

**Review by danieldiekmeier — APPROVED** [review]

File: ui/layouts/MobileApp.tsx (hunk 1 above)
Comment (endverbraucher) [reply ui/layouts/MobileApp.tsx:80]: Hmm. Good question! Technically it's not that easy. The logo element is also used on '/my-checkins' and is fixed positioned there. For the moment I would like not to change this if not really necessary. But we need a more stable way of displaying logos when we get a general whitelabeling solution.

File: pages/business/index.tsx (hunk 7 above)
Comment (endverbraucher) [reply pages/business/index.tsx]: 🤦

**Pull request merged**